### ISSUE TITLE
Regression in v92: set_child removed

### ISSUE THREAD

**Issue by vanvugt** [micheleg/dash-to-dock#2211]
It looks like there's a regression in 6745d8cc5b7ff21eb09dac46f74865be8019067b which reverted the fix of 42ee745074e9891e87cc7fa072797a15e9c72e98. `set_child` should be used in place of `add_child` to avoid those crashes.

CC @3v1n0 @sergio-costas

**Comment by sergio-costas** [comment]
@vanvugt I did that change because the window preview (middle-click in an app with several windows) didn't work. I tried to revert that using set_child, but either it fails (no window thumbnails are shown in the popup), or directly a javascript error is shown in the log because the widget doesn't have set_child(). The only place where using set_child() seems to work is at line 362, where the "close-button-symbolic" icon is placed over each thumbnail.

**Comment by vanvugt** [comment]
> The only place where using set_child() seems to work is at line 362, where the "close-button-symbolic" icon is placed over each thumbnail.

Yes that sounds like what I found when working on the original GNOME 45 bug. Can we check if it's regressed?
https://bugs.launchpad.net/bugs/2056506
